### PR TITLE
CODENVY-619 Restrict usernames to letters and digits only

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Normalize usernames that are created with OAuth, so they don't contain illegal characters.

### CODE REVIEW

File: core/codenvy-hosted-sso-auth-bearer/src/main/java/com/codenvy/auth/sso/server/BearerTokenAuthenticationService.java [hunk 1]
```diff
@@ -127,6 +129,9 @@ public Response authenticate(@CookieParam("session-access-key") Cookie tokenAcce
         Map<String, String> payload = handler.getPayload(credentials.getToken());
         handler.authenticate(credentials.getToken());
         final String username =  payload.get("username");
+        if (isValidUserName(username)) {
+
```
Comment (sleshchenko) [inline core/codenvy-hosted-sso-auth-bearer/src/main/java/com/codenvy/auth/sso/server/BearerTokenAuthenticationService.java]: are you sure that you don't need to do anything here?

Comment (mkuznyetsov) [inline core/codenvy-hosted-sso-auth-bearer/src/main/java/com/codenvy/auth/sso/server/BearerTokenAuthenticationService.java]: fixed

**Comment by skabashnyuk** [comment]
Any tests?

**Comment by skabashnyuk** [comment]
ok

**Comment by skabashnyuk** [comment]
ok

**Comment by sleshchenko** [comment]
LGTM

**Pull request merged**